src/conversations/Conversations.ts

```diff
@@ -185,15 +186,7 @@ export default class Conversations {
     const out: ConversationV2[] = []
     for (const response of responses) {
       try {
-        let convo = this.saveInviteResponseToConversation(response)
-
-        const initialMembers = convo.context?.metadata.initialMembers.split(',')
-
-        if (initialMembers && initialMembers.length > 0) {
-          convo = GroupConversation.from(convo, initialMembers)
```

Comment:
Don't we still need this code?

Reply:
This is unnecessary now that we have group invite topics.